### PR TITLE
Softmerg to dynamic menu

### PR DESCRIPTION
**Pull request by niewinny**

### CODE REVIEW

**Comment by mx1001** [comment]
now the menu doesnt show when nothing is selected.

**Comment by mx1001** [comment]
soft merge also isnt in the list!

**Comment by mx1001** [comment]
merge is gone.

File: gui_menus.py [hunk 1]
```diff
@@ -51,6 +51,17 @@ def draw(self, context):
...
             return
 
+
+        selected = bpy.context.selected_objects
+        active = context.active_object
+        selected.remove(active)
+        try:
+            object = selected[0]
+        except IndexError:
+            object = context.active_object
```
Comment (niewinny) [inline gui_menus.py]: those lines are supposed to be used to detect selection, but it produces problem if nothing is selected...
not sure how to go about it

**Comment by mx1001** [comment]
the edit menu for some reason wont show haha. What a disaster lol. I pray for jacque to look at it.

**Comment by niewinny** [comment]
ok I done it lol I was so tiered yesterday I am not sure why I didn't figure it out

**Comment by niewinny** [comment]
You can merge that if You wanna and I will start to rebuilding the menu file next, since I think I get the idea how to do it

**Comment by mx1001** [comment]
ill test it first
lets see if i can trust you. Im so excited about the using merges before merging!

**Pull request merged**